Loader/SymfonyFixturesLoader.php

```diff
@@ -21,14 +21,17 @@
 {
     private $loadedFixtures = [];
 
+    private $setsFixtureMapping = [];
+
     /**
      * @internal
      */
     public function addFixtures(array $fixtures)
     {
         // Store all loaded fixtures so that we can resolve the dependencies correctly.
         foreach ($fixtures as $fixture) {
-            $this->loadedFixtures[get_class($fixture)] = $fixture;
+            $this->loadedFixtures[get_class($fixture['fixture'])] = $fixture['fixture'];
```

Comment:
Instead of repeating `get_class($fixture['fixture'])]` twice, would it not be better to put it in a variable?